eurocodedesign/geometry/steelsections/__init__.py

```diff
@@ -2,17 +2,19 @@
 STEEL PROFILE CLASSES
 """
 import os
-import pandas as pd
+from typing import Any
 
-from pathlib import Path
 from dataclasses import dataclass
+import pandas as pd
+from pathlib import Path
```

Comment:
`dataclasses` and `pathlib` should go in the first block as they are standard library imports